app/services/hyrax/article_genre_service.rb

```diff
@@ -0,0 +1,9 @@
+# based on Hyrax::RightsStatementService
+module Hyrax
```

Comment:
Should we prefer the `NewspaperWorks` namespace here?  I am unclear if using the `Hyrax` namespace is required, or just conventional per documentation?

Reply:
I actually tried to put this within the `NewspaperWorks` namespace first, but I couldn't get it working properly.

After changing it to `Hyrax` it "just worked." I probably could have spent more time figuring out _why_, but decided to move on for the sake of time.

Reply:
"Just worked" sounds good to me, vs. effort on something relatively cosmetic like this.